core/authority-discovery/primitives/Cargo.toml

```diff
@@ -1,15 +1,15 @@
 [package]
-name = "substrate-consensus-common-primitives"
+name = "substrate-authority-discovery-primitives"
```

Comment:
this file has been moved, not copied to a new dir

Reply:
@rphmeier I don't understand this comment. Can you add some more details? 

This pull request introduces `substrate-authority-discovery-primitives` and removes `substrate-consensus-common-primitives`. Instead of showing the two operations separately, Git interprets them as a *move*.

Reply:
yeah. why did you remove that?

Reply:
Discussion happened offline:

`core/consensus/common/primitives` was introduces in https://github.com/paritytech/substrate/pull/3207. It only contained the `ConsensusApi`. `ConsensusApi` is obsolete with this pull request and is called nowhere. Thereby this pull request removes `core/consensus/common/primitives` entirely.